### ISSUE TITLE
List the container image location in the Readme

### ISSUE THREAD

**Issue by mulbc** [observatorium/token-refresher#25]
You have a Dockerfile in your repository. Are you pushing new images somewhere? If so, please mention where in the readme.

**Comment by squat** [comment]
Yes, that sounds like a good idea. I actually had to look it up myself to confirm where we store the image:
https://github.com/observatorium/token-refresher/blob/master/Makefile#L17
